scss/_labels.scss

```diff
@@ -43,9 +43,13 @@ a.label {
 .label-pill {
   padding-right: .6em;
   padding-left: .6em;
-  border-radius: 1rem;
 }
 
+@if $enable-rounded {
```

Comment:
Use the mixin please:
https://github.com/twbs/bootstrap/blob/807fb8847aa8cbe54ad371fccdec93b2518c1558/scss/mixins/_border-radius.scss#L3